R/formula-tools.R

```diff
@@ -512,41 +528,44 @@ construct_re <- function(re, data) {
     # and without rhs interactions
     for (k in j) {
       if (length(k) == 1) {
-          if (terms_int[i]) {
-            effects <- enw_add_pooling_effect(
-              effects, var_name = gsub(":", "__", k, fixed = TRUE),
-              finder_fn = function(effect, pattern) {
-                grepl(pattern[1], effect) &
-                  grepl(pattern[2], effect, fixed = TRUE) &
-                  lengths(
-                    regmatches(effect, gregexpr(":", effect, fixed = TRUE))
-                  ) == 1
-              },
-              pattern = strsplit(k, ":", fixed = TRUE)[[1]]
-            )
-          } else {
-            effects <- enw_add_pooling_effect(
-              effects, var_name = k,
-              finder_fn = function(effect, pattern) {
-                grepl(pattern, effect) & !grepl(":", effect, fixed = TRUE)
-              },
-              pattern = k
-            )
-          }
+        if (terms_int[i]) {
```

Comment:
unclear why this should be changing - doesn't actually appear to be, but I'd also rather not be doubling checking that it isn't.

Reply:
Aaah I think this was my auto-linter changing the indentation... Can revert